### PR TITLE
Add "Upcoming Events" widget.

### LINKED ISSUE #65: Feature Request--Link to gmail calendar
Copied from https://github.com/osu-cass/cass-pelican/issues/34:

> Carlos suggested that we put the upcoming events/conferences/meetings into a google calendar that links to the home page.
> Like the one in this site: http://surplus.oregonstate.edu/
> So that the current events stay current.

### PR DESCRIPTION
**Pull request by LyonesGamer**
Fixes #65.

Add a widget to the site which will list upcoming events from the CASS Website Calendar.

### CODE REVIEW

File: templates/includes/calendar-widget.html [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+// Your Client ID can be retrieved from your project in the Google
+// Developer Console, https://console.developers.google.com
+var CLIENT_ID = '';
+
+var SCOPES = [
+    "https://www.googleapis.com/auth/calendar.readonly"
+];
+
+var CALENDAR_ID = 'oregonstate.edu_s37pfj8sngtqk8s9fj66pslji8@group.calendar.google.com';
```
Comment (ramereth) [inline templates/includes/calendar-widget.html]: Can this please be set within the pelicanconf.py?

**Comment by ramereth** [comment]
@LyonesGamer that looks better but I'm not sure what I should be seeing when I test this. Are there any calendar entries already in it?

**Comment by LyonesGamer** [comment]
You need to get an API Key and OAuth Client ID from the Google Calendar API for the website (https://console.developer.google.com/), and put those in the constants in calendar-widget.html, and make sure the calendar ID ``oregonstate.edu_s37pfj8sngtqk8s9fj66pslji8@group.calendar.google.com`` is set in pelicanconf.py. Then it should display all the events in the CASS Website Calendar.

**Comment by ramereth** [comment]
@LyonesGamer where is that going to be set for each site that uses this feature?

**Comment by LyonesGamer** [comment]
Are they going to each have a different pelicanconf? Otherwise, how are different configurations set for each site?

**Comment by ramereth** [comment]
They will each have their own pelicanconf.

**Comment by LyonesGamer** [comment]
I moved the client ID and auth key into the pelicanconf as well so each site can set their own.

**Comment by ramereth** [comment]
@LyonesGamer and how do I generate said keys on their site?

**Comment by ramereth** [comment]
@LyonesGamer yes, please put that in the README. How many events will it show?

**Comment by LyonesGamer** [comment]
Up to 10. That can be reduced though. Should I also put that in the conf?

**Comment by ramereth** [comment]
@LyonesGamer yes, I would put that in the conf. I think 10 might be too much. Maybe three? Also, does it provide a link to the calendar so folks can see it in full?

**Comment by ramereth** [comment]
Unfortunately no longer needed as the cass website is moving to Drupal hosted by EECS.

**Pull request closed without merging**